assets/javascripts/kitten/helpers/utils/local-storage.test.js

```diff
@@ -0,0 +1,115 @@
+import { expect } from 'chai'
+import { localStorageUtils } from 'kitten/helpers/utils/local-storage'
+import sinon from 'sinon'
+
+describe('set()', () => {
+  const key = 'alice'
+  const value = 'wonderland'
+
+  before(() => {
+    localStorageUtils.set(key, value)
+  })
+
+  it('add a entry in localStorage', () => {
+    const length = Object.keys(localStorage).length
+
+    expect(length).to.be.equal(1)
+  })
+
+  it('affects a JSON value to the specified key', () => {
+    const fetched = localStorage.getItem(key)
+    const expected = JSON.stringify(value)
+
+    expect(fetched).to.be.equal(expected)
+  })
+
+  describe('when localStorage is not defined', () => {
+    const originalLocalStorage = localStorage
+
+    before(() => {
+      localStorage = undefined
+    })
+
+    it('returns null', () => {
+      const fetched = localStorageUtils.get('someKey')
+
+      expect(fetched).to.be.null
+    })
+
+    after(() => {
+      localStorage = originalLocalStorage
+    })
+  })
+})
+
+describe('get()', () => {
+  describe('with localStorageUtils', () => {
```

Comment:
Je trouve l'intitulé un peu étrange "get sans localStorageUtils" alors que le get est sur localStorageUtils. Ce ne serait pas plutôt `when setting the value with localStorageUtils` que tu voulais dire ici ?